backend/dal/Migrations/20210206053021_v01.07.02.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using Pims.Dal.Helpers.Migrations;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010702 : SeedMigration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
I thought we were going to add `Property.PropertyType`, which would result in both the `Parcel.PropertyType` and `Building.PropertyType`.  I suppose in some ways it's an imperfect relationship, as some property types don't apply to either building or parcel.